testing/trino-product-tests-launcher/src/main/resources/docker/trino-product-tests/conf/environment/multinode-apache-ranger/ranger-trino-security.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<configuration xmlns:xi="http://www.w3.org/2001/XInclude">
+    <property>
+        <name>ranger.plugin.trino.service.name</name>
+        <value>dev_trino</value>
+        <description>
+            Name of the Ranger service containing policies for this SampleApp instance
+        </description>
+    </property>
+
+    <property>
+        <name>ranger.plugin.trino.policy.rest.url</name>
+        <value>http://host.docker.internal:6080</value>
+        <description>URL to Ranger Admin</description>
+    </property>
+
+    <property>
+        <name>ranger.plugin.trino.policy.cache.dir</name>
+        <value>/tmp</value>
```

Comment:
Maybe let's use more specific location. `/tmp` may have other files that may maybe break Ranger in future.

Reply:
@kokosing  - `trino-policies.json` is the only file Ranger product test setup copies into this location (`/tmp`), which the tests load during execution. I was looking to use a directory like `/tmp/apache-ranger-product-tests`, but couldn't find a way to create a directory in docker container. If there is a better location to use?

Reply:
You could use something like https://github.com/trinodb/trino/blob/b40ce570b71b82a983b4cd4b20b8e6b6924618a6/testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/HadoopKerberos.java#L75, where you can have a script to create such directory. 

Anyway, it is not much important comment. Feel free to ignore it.